app/models/application.rb

```diff
@@ -5,14 +5,7 @@ class Application < ApplicationRecord
   PROJECT_VISIBILITY_WEIGHT = ENV['PROJECT_VISIBILITY_WEIGHT'] || 2
   COACHING_COMPANY_WEIGHT = ENV['COACHING_COMPANY_WEIGHT'] || 2
   MENTOR_PICK_WEIGHT = ENV['MENTOR_PICK_WEIGHT'] || 2
-  FLAGS = [:remote_team,
-          :volunteering_team,
-          :selected,
-          :male_gender,
-          :zero_community,
-          :age_below_18,
-          :less_than_two_coaches,
-          :less_than_40_hours_a_week]
+  FLAGS = Selection::Table::FLAGS + [:selected]
```

Comment:
Good job 👍 

To be honest, I fear that skipping the `selected` flag on the table was probably a mistake - I think it's fine to just use the exact same collection in both cases 🤞

Reply:
I'm not sure. I'm all for merging the flags, and adding the `:selected` one to the selection table won't break anything, but I wonder if it makes sense, as selected teams would never show up during the selection process, wouldn't they?

Reply:
I don't remember the exact details of the selection process. But I'd say they indeed need to show up in the table as I don't see another way for reviewers to have an overview on which teams were selected.

The actual "reviews" are based on each reviewer's `todos` which is a second list view. The table shows all the teams with their points, review status etc. and is then the place where teams will be selected.

Reply:
That makes sense! I'll make this change. Thanks for the feedback